### DISCUSSION TITLE
cannot join discord

### DISCUSSION THREAD

**Discussion by Vandddy** [zkSync-Community-Hub/zksync-developers#157] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I am working in zksync for months but i still cannot join discord

### What is the specific issue or error you're encountering?

I get the " unable to accept invite" in discord.

### Can you share the error messages or logs you're receiving, if any?

unable to accept invite

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

donot know

### Have you tried to isolate the problem, and if so, what were the results?

nothing

### What steps have you already taken to try to resolve the issue?

i tried from different links

**Comment by dutterbutter** [comment]
Can you assist here @bxpana

**Comment by bxpana** [comment, marked as answer]
@Vandddy thanks for reaching out. To make sure we keep the Github Discussion focused on developer support, you can email us at support@zksync.io and we can help you there. Thanks!
